toys/release/perform.rb

```diff
@@ -260,7 +260,11 @@ def publish_rad dry_run: false
         return
       end
       logger.info "**** Starting publish_rad for #{gem_name}"
-      cli.run(*tool_name[0..-2], "build-rad", "--gem-name", gem_name, "--friendly-api-name", friendly_api_name)
+      result = cli.run(*tool_name[0..-2], "build-rad", "--gem-name", gem_name, "--friendly-api-name", friendly_api_name)
+      unless result.zero?
+        logger.error "**** build-rad failed! Aborting publish_rad."
```

Comment:
Don't we want to log the result as well in case of an error?

Reply:
The subprocess will have logged the error. At this point here all we have is the process result code (zero or nonzero).